uint8 agent_number
std_msgs/Float64MultiArray box
